package.json

```diff
@@ -1,21 +1,34 @@
 {
-  "name": "orchardcore",
+  "name": "@orchardcore/root",
   "private": true,
+  "version": "1.0.0",
+  "packageManager": "yarn@4.6.0",
   "scripts": {
-    "build": "gulp build",
-    "rebuild": "gulp rebuild",
-    "watch": " gulp watch",
-    "postinstall": "node .scripts/install.js",
-    "update-projects": "node .scripts/update.js",
-    "install-project": "node .scripts/project.js install",
-    "update-project": "node .scripts/project.js update",
-    "cleanup": "node .scripts/cleanup.js"
+    "gulp-build": "gulp build",
+    "gulp-watch": "gulp watch",
+    "gulp-rebuild": "gulp rebuild",
+    "build": "assets-manager build",
+    "host": "assets-manager host",
+    "copy": "assets-manager copy",
+    "watch": "assets-manager watch",
+    "dry-run": "assets-manager dry-run",
+    "clean": "assets-manager clean",
+    "lint": "eslint ."
```

Comment:
Should this work?

![{FC68EB30-DDF9-4B46-BCB0-924C863D2B88}](https://github.com/user-attachments/assets/bc6a198c-bdcc-47b0-b9af-924c5e77312f)

Reply:
Yeah, I see :|.